### PR TITLE
allow-button-to-act-as-block-and-fill-up-cy-480

### PR DESCRIPTION
**Pull request by steiningermona**
replacing any type for styleoverwrite with string works fine

### CODE REVIEW

File: src/components/button/Button.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ export interface IButtonProps {
   htmlType?: string;
   activated?: boolean;
   sticked?: boolean;
+  styleOverwrite?: any;
```
Comment (diegomart2000) [inline src/components/button/Button.tsx]: Please review type

**Review by diegomart2000 — CHANGES_REQUESTED** [review]

File: src/components/button/Button.tsx (hunk 1 above)
Comment (steiningermona) [reply src/components/button/Button.tsx]: @diegomart2000 I copied the type for the overwrite from the other components where the styleOverwrite was used. What should I replace it with?

Comment (diegomart2000) [reply src/components/button/Button.tsx]: @steiningermona make it a string, css should be a string unless this causes troubles. If that's the case, we will look for a better type. Let me know.

Comment (steiningermona) [reply src/components/button/Button.tsx]: @diegomart2000 I tested it out with type string and it worked

**Review by diegomart2000 — APPROVED** [review]

**Pull request merged**